wpilibc/src/main/native/cpp/Watchdog.cpp

```diff
@@ -141,12 +146,14 @@ void Watchdog::PrintEpochs() {
   if (now - m_lastEpochsPrintTime > kMinPrintPeriod) {
     m_lastEpochsPrintTime = now;
     for (const auto& epoch : m_epochs) {
-      wpi::outs() << '\t' << epoch.getKey() << ": "
-                  << wpi::format(
-                         "%.6f",
-                         duration_cast<microseconds>(epoch.getValue()).count() /
-                             1.0e6)
-                  << "s\n";
+      wpi::SmallString<128> buf;
+      wpi::raw_svector_ostream err(buf);
+      err << '\t' << epoch.getKey() + ": "
+          << wpi::format(
+                 "%.6f",
+                 duration_cast<microseconds>(epoch.getValue()).count() / 1.0e6)
+          << "s\n";
+      frc::DriverStation::ReportWarning(err.str());
```

Comment:
The behavior here is different between C++ and Java.  In Java, ReportWarning is only called once (it's outside the loop body), in C++ it's called every loop iteration.  I think we want to only call it once.

Also, the call here doesn't need the `frc::` qualification.

Reply:
I did the Java version and @calcmogul gave me the C++ part, so that caused the difference.

Reply:
I know it is different, but it should not be.  Let's fix the C++ version to match Java.